core/client/client.go

```diff
@@ -12,32 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package grpcc is a client for communicating with the Key Server.  It wraps
-// the gRPC apis in a rpc system neutral interface and verifies all responses.
-package grpcc
+// Package client is a client for communicating with the Key Server.
+// It wraps the gRPC apis and verifies all responses.
+package client
 
 import (
 	"bytes"
 	"context"
-	"crypto"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"log"
 	"time"
 
-	"github.com/google/keytransparency/core/client/kt"
 	"github.com/google/keytransparency/core/crypto/signatures"
-	"github.com/google/keytransparency/core/crypto/vrf"
 	"github.com/google/keytransparency/core/crypto/vrf/p256"
 	"github.com/google/keytransparency/core/mutator"
 	"github.com/google/keytransparency/core/mutator/entry"
 
 	"github.com/google/trillian"
 	"github.com/google/trillian/client"
```

Comment:
It might be clearer to use an alias for this import, since it has the same name as this package. The name "tclient" seems to be used elsewhere.

Reply:
done